src/integrationTest/java/uk/ac/ebi/biosamples/cohortatlas/repository/CohortSearchRepositoryIntegrationTest.java

```diff
@@ -37,7 +40,13 @@ void findPage() {
     cohortService.saveCohort(cohort);
 
     Page<Cohort> page = cohortSearchRepository.findPageWithFilters(PageRequest.of(0, 5), "CohortSearchRepositoryIntegrationTest", "cohortName", Collections.emptyList());
-    Assertions.assertTrue(page.getSize() > 1);
-    Assertions.assertEquals("SearchRepositoryIntegrationTest_freeTextSearch_1", page.getContent().get(0).getCohortName());
+    assertTrue(page.getSize() > 1);
```

Comment:
Pleased use [assertj](https://joel-costigliola.github.io/assertj) for the tests. It would make tests readable.

Reply:
+1 @snathanvj could you please write tests for accession generation. 
I will use assertj for other test classes